src/buildpacks.rs

```diff
@@ -42,20 +44,78 @@ pub(crate) fn read_image_repository_metadata(
         .map(|value| value.to_string())
 }
 
-pub(crate) fn find_releasable_buildpacks(starting_dir: &Path) -> std::io::Result<Vec<PathBuf>> {
-    find_buildpack_dirs(starting_dir, &[starting_dir.join("target")]).map(|results| {
-        results
-            .into_iter()
-            .filter(|dir| dir.join("CHANGELOG.md").exists())
-            .collect()
-    })
+pub(crate) fn find_releasable_buildpacks(
+    starting_dir: &Path,
+) -> Result<Vec<PathBuf>, FindReleasableBuildpacksError> {
+    find_buildpack_dirs(starting_dir)
+        .map(|results| {
+            results
+                .into_iter()
+                .filter(|dir| dir.join("CHANGELOG.md").exists())
+                .collect()
+        })
+        .map_err(|e| FindReleasableBuildpacksError(starting_dir.to_path_buf(), e))
+}
+#[derive(Debug)]
+pub(crate) struct FindReleasableBuildpacksError(PathBuf, ignore::Error);
+
+impl Display for FindReleasableBuildpacksError {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let path = &self.0;
+        let error = &self.1;
+        write!(
+            f,
+            "I/O error while finding buildpacks\nPath: {}\nError: {error}",
+            path.display()
+        )
+    }
+}
```

Comment:
Couldn't we use `thiserror` here? We wouldn't need to implement `Display` by hand (which requires a bunch of boilerplate as seen here) and we would also get [std::error::Error](https://doc.rust-lang.org/std/error/trait.Error.html) implementations "for free". This applies to all places where we added `Display` implementations.

Reply:
switched all the `Display` implementations for `thiserror`